doc/adr/0010-js-purge.md

```diff
@@ -0,0 +1,67 @@
+# 10. js-purge
+
+Date: 2021-06-30
+
+## Status
+
+Accepted
+
+## Context
+
+JetStream provides the ability to purge streams by sending a request message to:
+`$JS.API.STREAM.PURGE.<streamName>`. The request will return a new message with
+the following JSON:
+
+```typescript
+{
+    type: "io.nats.jetstream.api.v1.stream_purge_response", 
+    error?: {
+        code: number,
+        err_code: number,
+        description: string
+    },
+    success: boolean,
+    purged: number
+}
+```
+
+The `error` field in the response will only be set if there was an error
+processing the request. The `success` field will be set to `true` if the request
+succeeded. The `purged` field will be set to the number of messages that were
+purged from the stream.
```

Comment:
not sure every ADR has to explain this, since its how every single API response works and (in theory) covered by the overall api design ADR?